docs/docs/cmd/planner/roster/roster-remove.md

```diff
@@ -0,0 +1,47 @@
+# planner roster remove
+
+Removes a Microsoft Planner Roster
+
+## Usage
+
+```sh
+m365 planner roster remove [options]
+```
+
+## Options
+
+`i, --id <id>`
+: ID of the Planner Roster.
+
+`--confirm`
+: Don't prompt for confirmation.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+!!! attention
+    Deleting a Planner Roster will also delete all plans within the Roster.
```

Comment:
Just noticed this, later I found out a Planner Roster can contain only 1 plan. Could you update the comment to make it singular? This is totally my bad!

Reply:
No problem, can happen!